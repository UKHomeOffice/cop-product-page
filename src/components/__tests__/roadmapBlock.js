import React from "react";
import renderer from "react-test-renderer";
import RoadMapBlock from "../roadmap-block";


describe("roadmapBlock", () => {
  const text = 'roadmap item';
  it("renders correctly", () => {
    const tree = renderer
      .create(<RoadMapBlock text={text}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct RoadMapBlock 'text' prop passed in", () => {
    const tree = renderer
      .create(<RoadMapBlock text={text}/>);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty('text', text);
  });
});