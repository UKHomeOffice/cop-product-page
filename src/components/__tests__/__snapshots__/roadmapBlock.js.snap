// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`roadmapBlock renders correctly 1`] = `
<div
  className="rectangle rectangle-block"
  style={
    Object {
      "borderLeft": "solid 6pxundefined",
    }
  }
>
  <div
    className="roadmap-block-text"
  >
    roadmap item
  </div>
</div>
`;
